testing/regress/ecl/keydiff1.ecl

```diff
@@ -18,6 +18,8 @@
 import $.setup;
 prefix := setup.Files(false, false).QueryFilePrefix;
 
+// This analysis tool message (related to skew caused by child records) is misleading
```

Comment:
Pedantic, but it's not being suppressed here because it's misleading, it's being suppressed here because it depends on the size of the thor you run on, and we need the output to be consistent for the regression suite...